WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationCommentDetailViewController.swift

```diff
@@ -326,4 +354,19 @@ private extension NotificationCommentDetailViewController {
         static let imageName = "wp-illustration-notifications"
     }
 
+    // MARK: - Types
+
+    private enum Content {
```

Comment:
[Nit] Some documentation here would be great 🙏

Reply:
Resolved in 1e2d7b7362b4854571094d2106d6c3bfc3c87564